doc/openconfig_style_guide.md

```diff
@@ -256,11 +258,16 @@ identity LC_CONNECTOR {
 }
 ```
 
-
 ## YANG language usage
 Language rules describe guidelines on use of specific YANG language statements,
 including how modules should be structured and parsed.
 
+### `default`
+
+The use of default should be avoided.  Instead a leaf should be explicitly
```

Comment:
Can we also include the rationale for this recommendation? the details you have in this PR's description seems to be useful here as well

Reply:
Good idea, I've updated the text